exporter/exporterhelper/internal/queuebatch/queue_batch_test.go

```diff
@@ -514,47 +480,6 @@ func TestQueueBatch_DrainActiveRequests(t *testing.T) {
 	assert.Equal(t, 3, sink.ItemsCount())
 }
 
-func TestQueueBatchWithTimeout(t *testing.T) {
```

Comment:
I agree with removing this test. I assume other tests cover this behavior, it's not a very strong test condition.